src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/WindowProcess.cs

```diff
@@ -33,6 +33,14 @@ internal uint ProcessID
             get; private set;
         }
 
+        /// <summary>
+        /// Gets a value indicating whether the process is responding or not
+        /// </summary>
+        internal bool Responding
```

Comment:
```suggestion
        internal bool IsResponding
```

I think this is more clear and more in line with the other state related properties.

Reply:
Fixed in commit [674371e](https://github.com/microsoft/PowerToys/pull/33167/commits/674371e9221ebac3c80dcdc8c3156a7512be04d3)